docs/coronavirus_page_publishing_tool.md

```diff
@@ -0,0 +1,48 @@
+## Coronavirus Publishing Tool
+
+There are currently three coronavirus hub pages(/coronavirus/business-support, /coronavirus/education-and-childcare, /coronavirus/worker-support), and a coronavirus landing page. As we approached lockdown, a basic publishing interface was required at short notice. This is how it works:
+
+1. Content is added to the relevant yaml file in the [govuk-coronavirus-content](https://github.com/alphagov/govuk-coronavirus-content/tree/master/content) repository.
+2. Content editor with Coronavirus editor signon permissions visits [Coronavirus pages](https://collections-publisher.publishing.service.gov.uk/coronavirus) tab of collections publisher.
```

Comment:
I'd suggest a slight rewording to address two issues
- I don't think it's immediately clear that Content editor is a user
- The permission is defined as a specific string, which I think would helpful to use here. 

So I'd suggest:
```
2. Content editor (a user with "Coronavirus editor" permissions) visits [Coronavirus pages](https://collections-publisher.publishing.service.gov.uk/coronavirus) tab of collections publisher.
```